decks/slides/index.html

```
@@ -234,8 +232,8 @@ <h3 class="font-space tracking-tight">Engagement Model</h3>
           <h3 class="font-space tracking-tight">Pipeline</h3>
           <div class="mermaid-src">
 graph LR
-  D["Discovery Call"] --> S["Custom Setup"]
-  S --> R["Retainer"]
+  D["AI Workflow Audit"] --> S["Done-For-You Build"]
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)